app/templates/direct-award/end-search.html

```diff
@@ -62,7 +62,11 @@ <h2 class="heading-xmedium">After your search has ended</h2>
         method="POST"
         >
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-        <button class="button-save end-your-search-button">End search and continue</button>
+        <button class="button-save end-your-search-button"
+          {% if disable_end_search_btn %} 
+           disabled="disabled" aria-disabled="true" 
```

Comment:
Not important, but FYI https://stackoverflow.com/questions/38059140/when-to-use-the-disabled-attribute-vs-the-aria-disabled-attribute-for-html-eleme